indico/htdocs/js/indico/modules/registration/form/field.js

```diff
@@ -443,6 +431,32 @@ ndRegForm.directive('ndRadioField', function(url) {
         require: 'ndField',
         controller: function($scope) {
             $scope.tplInput = url.tpl('fields/radio.tpl.html');
+
+            $scope.initFormData = function(formData, field) {
+                formData.choices = [];
+                _.each(field.choices, function(item, ind) {
+                    formData.choices[ind] = angular.copy(item);
+                });
+            };
+
+            $scope.validateFieldSettings = function(dialogScope) {
+                if (!dialogScope.hasRadioItems()) {
+                    dialogScope.$apply(dialogScope.setSelectedTab('tab-editItems'));
```

Comment:
setSelectedTab doesn't return anything. Why not call `$apply` in a separate line?